src/vm/mscorlib.h

```diff
@@ -828,9 +827,6 @@ DEFINE_METHOD(INTERLOCKED,          COMPARE_EXCHANGE_OBJECT,CompareExchange, SM_
 DEFINE_CLASS(PINNING_HELPER,        CompilerServices,       PinningHelper)
 DEFINE_FIELD(PINNING_HELPER,        M_DATA,                 m_data)
 
-DEFINE_CLASS(ARRAY_PINNING_HELPER,  CompilerServices,       ArrayPinningHelper)
-DEFINE_FIELD(ARRAY_PINNING_HELPER,  M_ARRAY_DATA,           m_arrayData)
```

Comment:
Removing this is causing CI errors. Also it is used [here](https://github.com/dotnet/coreclr/pull/10988/files#diff-bbced44286fd2ad6bcf70fbd3c276d57R6926).

d:\j\workspace\debug_windows---17180f1b\src\vm\jitinterface.cpp(6926): error C2065: 'FIELD__ARRAY_PINNING_HELPER__M_ARRAY_DATA': undeclared identifier [D:\j\workspace\debug_windows---17180f1b\bin\obj\Windows_NT.x64.Debug\src\vm\crossgen\cee_crossgen.vcxproj]